src/ApiCall.ts

```diff
@@ -720,7 +785,7 @@ export async function generateBarcode(productId: number, params: {
 }
 
 // verify phone number
-export async function verifyPhoneNumber(phoneNumber: string): Promise<any> {
+export async function verifyPhoneNumber(phoneNumber: string, userID = undefined) {
```

Comment:
We don't need userID for this method. I just pushed a commit before fixing this as was raised by Viet. It should be in sendTwoFactor, but I've already tidied it up. Can merge that in to be up-to-date